src/bootstrap/server.bootstrap.js

```diff
@@ -38,6 +41,9 @@ export class Server {
     this.app.use("/health", healthRouter);
     this.app.use("/events", eventRouter);
     this.app.use("/user", userRouter);
+    this.app.use("/my", myRouter);
```

Comment:
Why did we add this? What functionality does it cover?

Reply:
It was intended to _handle_ requests for a specific user (i.e. the owner of certain events). But, as noted by @sonia-cristea we could solve this _problem_ in a simpler way.